administrator-manual/en/accounts.rst

```diff
@@ -248,6 +247,7 @@ account is erased.
     /var/lib/nethserver/home
     /var/lib/nethserver/vmail
     /var/lib/nethserver/ibay
+    /var/lib/nethserver/nextcloud
```

Comment:
I'm not sure that this should be here: nextcloud dir names are based on uuid and it's hard to retrieve the corresponding user.

Reply:
The section header is "Local accounts provider uninstall". 

I think it requires removing everything under that dir.

Reply:
No, because admin data are still there.
This is the same for any users not coming from the account providers.

Also the dir store the logs: it should not be deleted manaually

Reply:
I understand your point. However by your reply I feel there's something odd with ibay and vmail: there we have admin and shared data and users not coming from account providers too!

Back to the list is explanation, in the line above:

    This is the list of system directories containing users and groups data:

It does not state anything about subdir names, so do not consider my previous comment, when I wrote "removing everything". The list is just the place where users and groups data is stored, and removal is not automatic.

I'm still for adding nextcloud to that list, and also any similar directory I may forget.

Reply:
Ok, fine.

Reply:
I think we have to fix the whole section because the directory list with users and groups data is the same, no matter the account provider is.

The sentence "A list of users and groups in TSV format is dumped" is true for local providers though.